### PR TITLE
Fixes #494 : Remove all whitespace character when parsing jdbc url

### LINKED ISSUE #494: [Bug report] P6Spy : URL with white space makes parseServerAddress silently crash
URI.create crashes when the given String url contains white space : https://github.com/hypnoce/brave/commit/2cde7291d7b724b959961bea9a6e5323c11f5c78

Using the Microsoft SQL Server jdbc driver, the DatabaseMetadata#getUrl returns a url containing whitespace. Therefore, using this driver, we will miss some information regarding the endpoint.

@adriancole I will provide with a fix soon

### PR DESCRIPTION
**Pull request by hypnoce**

### CODE REVIEW

**Comment by hypnoce** [comment]
I'm not sure about replacing white space, but it seems the best alternative.

File: instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
   @Mock Span span;
   String url = "jdbc:mysql://127.0.0.1:5555/mydatabase";
+  String urlWithWhiteSpace = url + ";parameter=with white space;foo=bar";
```
Comment (adriancole) [inline instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java]: Can you use a realistic example of a parameter with whitespace?

Comment (hypnoce) [reply instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java]: Sure !

Comment (hypnoce) [reply instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java]: Done !

**Comment by hypnoce** [comment]
Weird failure in the grpc tests. Relaunching the build.

**Comment by adriancole** [comment]
great stuff

**Pull request merged**